medicines/README.md

```diff
@@ -1,33 +1,26 @@
-# Medicines Information Portal
+# MHRA Products Portal
```

Comment:
This area of the repo is named differently elsewhere, notably in the top-level `README.md`, where it's still called _Medicines Information Portal_. If the official name is _MHRA Products Portal_, can we change the top-level `README.md` to reflect that?

Reply:
Fixed! Made consistently "MHRA Products Portal"